docs/components/importdoc.vue

```diff
@@ -129,7 +132,7 @@ export default {
     pluginImportCode() {
       const pluginLocation = this.isComponentRoute ? 'components' : 'directives'
       return [
-        `import { ${this.pluginName} } from 'bootstrap-vue/es/${pluginLocation}'`,
+        `import ${this.pluginName} from 'bootstrap-vue/es/${pluginLocation}/${this.pluginDir}'`,
```

Comment:
Please add the brackets ` { ${this.pluginName} }` back here.

Reply:
Are you sure about this? It was actually Troy who removed this in 8af2743f90abc189a4269b1bba4a471020fbdafa as he added the `${this.pluginDir}` which has a default install export. Probably after my question yesterday as this syntax works better with tree shaking.